### PR TITLE
Add curl example to documentation

### PR DESCRIPTION
**Pull request by jzbahrai**
# Summary | Résumé

Add CURL example to documentation.

ToDO: French translation

### CODE REVIEW

**Review by whabanks — APPROVED** [review]

File: src/en/send.md [hunk 1]
```diff
@@ -516,3 +516,19 @@ If the request is not successful, the response body is `json`, refer to the tabl
...
 |`400`|`[{`<br>`"error": "BadRequestError",`<br>`"message": "Some rows have errors. Row 1 - name: Missing. Row 2 - email address: invalid recipient. Row 3 - name: Missing. Row 4 - name: Missing."`<br>`}]`|Make sure rows don't have missing values|
 |`500`|`[{`<br>`"error": "Exception",`<br>`"message": "Internal server error"`<br>`}]`|GC Notify was unable to process the request. Re-send your notification.|
+
+## CURL Example
+
+Example to send a single notification
+
+```
+curl --request POST \
+  --url https://notification.canada.ca/v2/notifications/email \
```
Comment (andrewleith) [inline src/en/send.md:526]: This url is incorrect.
```suggestion
  --url https://api.notification.canada.ca/v2/notifications/email \
```

File: src/en/send.md [hunk 2]
```diff
@@ -516,3 +516,19 @@ If the request is not successful, the response body is `json`, refer to the tabl
...
+  --header 'Authorization: ApiKey-v1 YOUR_API_KEY' \
+  --header 'Content-Type: application/json' \
+  --header 'base_url: https:///notification.canada.ca' \
+  --data '{
+        "email_address": "EMAIL_ADDRESS_TO_SEND_NOTIFICAITION",
+        "template_id": "TEMPLATE_ID"
+}
```
Comment (andrewleith) [inline src/en/send.md:533]: You are missing a single quote here.
```suggestion
}'
```

**Review by andrewleith — CHANGES_REQUESTED** [review]
Example looks good just a typo in the API url and a missing quote for the JSON data.

**Review by andrewleith — APPROVED** [review]
LGTM
:shipit:

**Pull request merged**